src/modules/nodes/nodeQueries.ts

```diff
@@ -162,7 +163,14 @@ const fetchChildNodesWithArticleType = async ({
   articleType?: string;
   isPublished?: boolean;
 })[]> => {
-  const childNodes = await fetchChildNodes({ id, taxonomyVersion, language, recursive: true });
+  const childNodesWithResources = await fetchChildNodes({
+    id,
+    taxonomyVersion,
+    language,
+    recursive: true,
+  });
+  const childNodes = childNodesWithResources.filter(x => x.nodeType !== RESOURCE_NODE);
```

Comment:
Antar det vil være kjappere å sende inn TOPIC og SUBJECT i spørringa, heller enn å hente ut alt og så filtrere ut RESOURCE.

Reply:
Da må vi utvide taxonomy til å støtte det for barn også. Vet ikke hvor vanskelig den rekursive biten av det er jeg(?)

Reply:
Godt poeng. Kan la det stå slik.